ui/push-health/CommitHistory.jsx

```diff
@@ -55,94 +61,126 @@ class CommitHistory extends React.PureComponent {
       repo: currentRepo.name,
     });
     const { author, push_timestamp: pushTimestamp } = currentPush;
-    const authorPushFilterUrl = getJobsUrl({ author, repo: currentRepo.name });
+    const headerText = revisions[0].comments.split('\n')[0];
+    const authorMatch = author.match(/<(.*?)>+/);
+    const authorEmail = authorMatch ? authorMatch[1] : author;
+    const expandIcon = isExpanded ? faCaretUp : faCaretRight;
+    const expandTitle = isExpanded ? 'Click to collapse' : 'Click to expand';
+    const expandText = isExpanded ? 'Hide all commits' : 'Show more commits';
```

Comment:
So, this is a bit of a nit-pick, but I wonder if instead of saying "Hide all commits" we used the text: "Hide commits" ?  The reason is that it doesn't actually hide all the commits.  It collapses it down to showing just 1.  :)